rubric_visualization/settings/base.py

```diff
@@ -195,6 +196,13 @@
 CANVAS_OAUTH_CANVAS_DOMAIN = CANVAS_DOMAIN
 CANVAS_OAUTH_CLIENT_ID = SECURE_SETTINGS.get('canvas_oauth_client_id')
 CANVAS_OAUTH_CLIENT_SECRET = SECURE_SETTINGS.get('canvas_oauth_client_secret')
+CANVAS_OAUTH_SCOPES = [
+    'url:GET|/api/v1/courses/:course_id/assignments',
+    'url:GET|/api/v1/courses/:course_id/users',
+    'url:GET|/api/v1/courses/:course_id/assignments/:assignment_id/submissions',
+    'url:GET|/api/v1/courses/:course_id/rubrics',
+    'url:GET|/api/v1/courses/:course_id/rubrics/:id',
+]
 
```

Comment:
So, this is essentially scoping the requests that we are allowed to make to the canvas api?

Reply:
Yeah, exactly. These limit the capability of the access tokens to just those scopes (the ones we are requesting).

One thing to keep in mind is that we can only request scopes that have been explicitly granted to our API key in Canvas. This is actually a subset of the scopes that have been granted tour our API key in the DEV environment.